src/StakeRegistry.sol

```diff
@@ -207,14 +207,36 @@ contract StakeRegistry is StakeRegistryStorage {
     }
 
     /// @notice Initialize a new quorum and push its first history update
-    function initializeQuorum(
+    function initializeDelegatedStakeQuorum(
+        uint8 quorumNumber,
+        uint96 minimumStake,
+        StrategyParams[] memory _strategyParams
+    ) public virtual onlyRegistryCoordinator {
+        require(!_quorumExists(quorumNumber), "StakeRegistry.initializeQuorum: quorum already exists");
+        _addStrategyParams(quorumNumber, _strategyParams);
+        _setMinimumStakeForQuorum(quorumNumber, minimumStake);
+        _setStakeType(quorumNumber, StakeType.TOTAL_DELEGATED);
+
+        _totalStakeHistory[quorumNumber].push(StakeUpdate({
+            updateBlockNumber: uint32(block.number),
+            nextUpdateBlockNumber: 0,
+            stake: 0
+        }));
+    }
+
+
+    /// @notice Initialize a new quorum and push its first history update
+    function initializeSlashableStakeQuorum(
```

Comment:
hm, feel like we should combine this with the other intialize and only do slashable/delegated discrimination inside this contract, not at the interface level